[HW] 1072589

Jhang-Jia-Jhen opened the pull request.

Commit:
[HW] 1072589

[欲解決問題]:autocrlf，因為Linux和windows系統上的差異，可能會出現[clone後status卻是修改過的紅字]或是[跳出warning: LF will be replaced by CRLF.]。

[解決辦法1]:以git config (--global) core.autocrlf 來修改預設值
[解決辦法2]:以Notepad++檢查是CRLF還是LF

Signed-off-by: 1072589 <s1072589@mail.ncyu.edu.tw>

jrjang commented:
Hi @Jhang-Jia-Jhen ,

1. 將修改加在 README.md 第 2 點, 用"提示"的方式表示
2. commit message 過於冗長，請重新撰寫
3. ex0.txt 不需要修改

jrjang commented on ex0.txt:
```diff
@@ -0,0 +1 @@
+MTA3MjU4OSxzMTA3MjU4OUBtYWlsLm5jeXUuZWR1LnR3Cg==
```
不需修改

jrjang commented on hw.txt:
```diff
@@ -0,0 +1,5 @@
+#查詢
```
hw.txt 可以刪除

jrjang commented on README.md:
```diff
@@ -11,6 +11,7 @@ Example 0:
 電子郵件: s1043000@mail.ncyu.edu.tw
 
 [注意] 如果已經有在使用 Git ，請先備份家目錄的 .gitconfig 檔案
+[注意] Linux與windows環境有別，建議以指令$git config --global core.autocrlf true 修改設定，或可先由指令$git config core.autocrlf 查詢預設值。
```
1. 中英文之間要有一個空白
2. "指令"或"程式碼"請套用 github Quoting code 格式 (https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

Jhang-Jia-Jhen replied:
原來README也有格式的! 學到了，非常感謝講師!

Commit:
[HW] 1072589

[欲解決問題]:autocrlf，因為Linux和windows系統上的差異，可能會出現[clone後status卻是修改過的紅字]或是[跳出warning: LF will be replaced by CRLF.]。

[解決辦法1]:以git config (--global) core.autocrlf 來修改預設值
[解決辦法2]:以Notepad++檢查是CRLF還是LF

Signed-off-by: 1072589 <s1072589@mail.ncyu.edu.tw>
